dw-form-field.js

```diff
@@ -8,16 +8,18 @@ Code distributed by Google as part of the polymer project is also
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 */
 
-import { css } from 'lit-element';
-import { Formfield } from '@material/mwc-formfield';
+import { css } from "lit-element";
+import { Formfield } from "@material/mwc-formfield";
+import { classMap } from "lit-html/directives/class-map";
+import { html, property, query } from "lit-element";
 
 export class DwFormField extends Formfield {
   static get styles() {
     return [
       Formfield.styles,
       css`
         :host {
-          display: block;
+          display: inline-block;
```

Comment:
Why display is changed to `inline-block`, we planned that we don't use `inline-block` in any new code, though?